src/components/featured-article/_macro.njk

```diff
@@ -0,0 +1,82 @@
+{% from "components/list/_macro.njk" import onsList %}
+{% from "components/chart/_macro.njk" import onsChart %}
+{% from "components/image/_macro.njk" import onsImage %}
+
+{% macro onsFeaturedArticle(params) %}
+    {% set headingLevel = params.headingLevel | default(2) %}
+    {% set openingArticleTitleTag = "<h" ~ headingLevel %}
+    {% set closingArticleTitleTag = "</h" ~ headingLevel ~ ">" %}
+
+    <div class="ons-featured">
+        {{ openingArticleTitleTag | safe }} class="ons-featured__title ons-u-fs-m ons-u-mt-no ons-u-mb-2xs">
+        <a href="{{ params.title.url }}">{{ params.title.text }}</a>
+        {{ closingArticleTitleTag | safe }}
+
+        {%- if params.metadata -%}
+            {% set metadata = params.metadata %}
+            <ul class="ons-featured__item-metadata">
+                {%- if metadata.date -%}
+                    <li class="ons-featured__item-attribute">
+                        {% set prefixClass = "ons-u-fw-b" if metadata.date.showPrefix == true else "ons-u-vh" %}
+                        <span class="{{ prefixClass }}">{{ metadata.date.prefix | default("Published") }}: </span>
+                        {%- if metadata.date.iso -%}
+                            <time datetime="{{ metadata.date.iso }}">{{ metadata.date.short }}</time>
+                        {%- endif -%}
+                    </li>
+                {% endif %}
+
+                {%- if metadata.object and metadata.object.text -%}
+                    <li class="ons-featured__item-attribute">
+                        {% set metadataObject %}
+                            <span {% if metadata.object.text %}class="ons-featured__attribute-text ons-u-fw-b"{% endif %}
+                                >{{ metadata.object.text }}</span
+                            >
+                        {% endset %}
+                        {{ metadataObject | safe }}
```

Comment:
Do we need `metadataObject` here if we are just going to set it and then output it straight after?